src/models/appointments.js

```diff
@@ -0,0 +1,28 @@
+const { Schema, model } = require('mongoose');
+
+const appointmentSchema = Schema(
+	{
+		date: {
+			type: Date,
+		},
+		hour: {
+			type: String,

```

Comment:
Why do u use "hour" in a database, if you can write the date with time in input "date" as UNIX value or 2022-08-12T13:02?
Source about it - https://www.mongodb.com/docs/manual/reference/method/Date/.

Reply:
As far as I understand we need a date and the time on this date.
So, why can't we use this feature of MongoDB?
<img width="673" alt="image" src="https://user-images.githubusercontent.com/63366123/189093789-b2f863c2-8d01-48e2-aa5c-eff5023841cf.png">

Reply:
It's a very good idea, because if we've already got the necessary functional from MongoDB (date ad time in one input), why do we go avoid of use it?